requirements/requirements.md

```diff
@@ -28,7 +29,7 @@ Notary v2 aims to address the learnings and gaps of v1, while prioritizing a set
 1. Key hierarchies and delegation
 1. Key revocation, including private and air-gapped registries
 1. Key acquisition must support users from hobbyists, open source projects to large software vendors
-1. Usable workflows, enabled for the masses to easily create and consume Notary v2 signatures
+1. Usable workflows, enabled for the masses to easily create and consume Notary signatures
```

Comment:
Usable workflows, enabled for the masses to easily create and consume Notary **Project** signatures